samples/amber.cc

```diff
@@ -64,6 +65,7 @@ const char kUsage[] = R"(Usage: amber [options] SCRIPT [SCRIPTS...]
   -e <engine>               -- Specify graphics engine: vulkan, dawn. Default is vulkan.
   -v <engine version>       -- Engine version (eg, 1.1 for Vulkan). Default 1.0.
   -V, --version             -- Output version information for Amber and libraries.
+  --verbose                 -- Log graphics API calls made by the Vulkan engine.
```

Comment:
--log-graphics-calls ? Which we then set into the delegate directly and return with the LogGraphicsCalls method?